packages/combobox/src/LionCombobox.js

```diff
@@ -2,6 +2,60 @@ import { html, css, browserDetection } from '@lion/core';
 import { OverlayMixin, withDropdownConfig } from '@lion/overlays';
 import { LionListbox } from '@lion/listbox';
 
+const matchReverseFns = new WeakMap();
+const matchA11ySpanReverseFns = new WeakMap();
+
+/**
+ * @param {Node} root
+ * @param {string} matchingString
+ * @param {object} option
+ */
+function checkForTextMatch(root, matchingString, option) {
```

Comment:
The name `checkForTextMatch` doesn't fully cover what it does anymore. Maybe call it `makeMatchingTextBold`
  
We can change it later into a util function that can be reused in contexts that need to traverse shadow dom (in which case root would have been option.shadowRoot).  For reference: see our demos.
Then we could call it checkForTextMatch again and allow a callback function (in case you want to do smth different than makling text bold.)